src/schedule/zephyr_domain.c

```diff
@@ -55,6 +56,14 @@ struct zephyr_domain {
 /* perf measurement windows size 2^x */
 #define CYCLES_WINDOW_SIZE	10
 
+#if CONFIG_WDT_DW
```

Comment:
just to make sure - this will be disabled with CONFIG_WATCHDOG=n?

Reply:
yes, it will